src/Commands/ModuleMakeCommand.php

```diff
@@ -35,9 +35,11 @@ public function handle()
                 ->setFilesystem($this->laravel['files'])
                 ->setModule($this->laravel['modules'])
                 ->setConfig($this->laravel['config'])
+                ->setActivator($this->laravel['modules.activator'])
```

Comment:
Let's try to avoid the alias key and instead use the class reference. I'm trying to move away from the aliases.